### DISCUSSION TITLE
Show checkboxes in Table

### DISCUSSION THREAD

**Discussion by kodecrash** [Budibase/budibase#6382] [Help] [answered]
I have configured REST API to display list of todos. I want to show checkboxes in a table for selecting multiple todos.

Here is my configuration. How I can enable checkboxes in table. I have also enabled allow row selection checkbox. But checkboxes are not visible.

![image](https://user-images.githubusercontent.com/8296994/174471354-b1bfba2f-0edd-4740-bc13-fa59f1fbe917.png)

**Preview.**

![image](https://user-images.githubusercontent.com/8296994/174471381-eaaf3d5f-c59b-4fe8-88e7-3497f0f0cd7c.png)

**Comment by melohagan** [comment, marked as answer]
Hi @kodecrash 

Row selection is not allowed for REST API data sources. https://github.com/Budibase/budibase/issues/4895#issuecomment-1157525518

Ideally that option should not be visible for REST API sources.

**Reply by kodecrash** [reply]
Thank you.
